### PR TITLE
Updated help command

### PR DESCRIPTION
**Pull request by jeweetzelfWasTaken**
read title

### CODE REVIEW

File: bot/discord/commands/help.js [hunk 1]
```diff
@@ -9,7 +9,11 @@ const commands = {
         uptime: "Shows the bots uptime.",
         vc: "Manage you custom voice channel.",
         info: "Get a bots info.",
-        animalapi: "Check out our animal api."
+        animalapi: "Check out our animal api.",
+        apikey: "Generates a(n) (new) API Key",
+        domains: "Shows all your proxied domains",
+        help: "Shows this help menu",
```
Comment (itzJOHv) [inline bot/discord/commands/help.js:15]:
```suggestion
        help: "Shows this menu.",
```

File: bot/discord/commands/help.js [hunk 2]
```diff
@@ -9,7 +9,11 @@ const commands = {
         uptime: "Shows the bots uptime.",
         vc: "Manage you custom voice channel.",
         info: "Get a bots info.",
-        animalapi: "Check out our animal api."
+        animalapi: "Check out our animal api.",
+        apikey: "Generates a(n) (new) API Key",
```
Comment (itzJOHv) [inline bot/discord/commands/help.js:13]:
```suggestion
        apikey: "Manage your API keys.",
```

File: bot/discord/commands/help.js [hunk 3]
```diff
@@ -9,7 +9,11 @@ const commands = {
...
+        domains: "Shows all your proxied domains",
```
Comment (itzJOHv) [inline bot/discord/commands/help.js:14]:
```suggestion
        domains: "Shows all your proxied domains.",
```

**Review by itzJOHv — CHANGES_REQUESTED** [review]

**Comment by jeweetzelfWasTaken** [comment]
Engrish is verrrrrrrry gut

**Pull request closed without merging**